demo/components/presentational/SellOrdersList.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { shallow } from "enzyme";
+import SellOrdersList from "./SellOrdersList";
+
+describe("SellOrdersList", () => {
+  it("renders the component", async () => {
+    expect(
+      shallow(<SellOrdersList sellOrders={[]} renderItem={() => {}} />)
```

Comment:
Might read more easily with the function that does nothing assigned to a var before this line

Reply:
Done.

Reply:
Thanks! Resolving.